cockpit-container-update

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+# Update .cockpit-ci/container to the latest tasks container tag automatically
+
+import json
+import os
+import socket
+import sys
+import urllib.parse
+import urllib.request
+from ssl import SSLEOFError
+
+import task
+from lib.constants import BASE_DIR
+
+sys.dont_write_bytecode = True
+
+
+URL = 'https://quay.io/api/v1/repository/cockpit/tasks/tag/?onlyActiveTags=true'
+
+
+def fetch_tags(url):
+    try:
+        req = urllib.request.Request(url)
+        with urllib.request.urlopen(req) as response:
+            return json.loads(response.read().decode().strip())
+    except (urllib.error.HTPPError, ConnectionResetError, socket.error, SSLEOFError) as e:
+        print("Transient error during quay.io request %s", e)
```

Comment:
Kindly borrowed this from our github.py code but maybe it is a bit over the top :)

Reply:
This should either be a retry loop and hard fail after a few retries, or just fail right away. The latter is fine IMHO -- we are going to run it every day anyway. Let's not silently ignore it, we won't be able to tell the difference between "the job didn't find something new" and "the job is broken".

TL/DR: Please drop all error handling here.

Reply:
Let's just fail and see how often it breaks.